.github/dependabot.yml

```diff
@@ -1,12 +1,12 @@
 version: 2
```

Comment:
I will remove this file. I think it's better to just bump deps before release, then accepting automatically created PR for each. WDYT?

Reply:
I think we can remove it :+1: I've added commit that removes it